data/gsettings/com.gexperts.Tilix.gschema.xml

```diff
@@ -1013,6 +1013,10 @@
       <default>'&lt;Ctrl&gt;&lt;Alt&gt;d'</default>
       <summary>Keyboard shortcut to add new terminal down</summary>
     </key>
+    <key name="session-add-auto" type="s">
+      <default>'&lt;Ctrl&gt;&lt;Alt&gt;x'</default>
```

Comment:
What about `Ctrl+Alt+a` as default (a for just "add")?

Reply:
Yes, fine by me. Should I change this in my pull request?

Reply:
Yes please